sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -4589,22 +4589,22 @@ pi_result cuda_piextUSMEnqueueMemcpy(pi_queue queue, pi_bool blocking,
   return result;
 }
 
+// flags is currently ignored as it is a placeholder for future features
```

Comment:
Very minor, but it is not "ignored" anymore, so maybe the comment should be inside the function and be adapted to reflect the new behavior for unrecognized flags.